pegasus/sites.v3/hourofcode.com/views/hoc_events_counter.haml

```diff
@@ -6,31 +6,15 @@
 -if @company
   -company_name = hoc_s("company_" + @company)
   -company_total = format_integer_with_commas(company_count)
-
-  %h2.counter-title
-    %a{href: resolve_url('/events/all')}
-      = events_text + ","
-      = hoc_s(:map_header_company).gsub('#', company_total).gsub('company', company_name)
-    -# Hide signup buttons during Hour of Code
```

Comment:
We used to show a "Sign up" button to the right of the map title. I was thinking of putting it under `hoc_mode` but it won't make sense to have a "Sign up" button after this text with this new design.